docs/debug_e2etests.md

```diff
@@ -0,0 +1,32 @@
+# Debugging e2etests
+Sometimes e2e tests fail and we need to figure out why.
+These tests will normally create a local kubernetes cluster where our operators will be installed and Custom Resources
+will be created to spin up tenant clusters on the tested provider.
+
+If we want to debug what happened, the first thing we would need to do would be to re-run the failed job in CircleCI
+enabling SSH so we can later connect to the CircleCI worker running the job. Click on the following button to do that:
+
+![](rerun.png)
+
+The job will be re executed, but this time there is a new step that let's you connect to the worker using SSH.
+
+![](ssh.png)
+
+## Inspecting control plane cluster
+Let's _ssh_ into the CircleCI worker to install `kubectl` so we can talk to our test cluster.
+Copy and run the command from this step:
+
+![](kubectl.png)
```

Comment:
It would be better to have this as a code block here - I don't want to write off that command from the picture or navigate there to copy it :P

Reply:
Yes, but I didn't want to hard-write the kubectl _version_, since it would be different depending on the minikube version.

Reply:
I kept the image _and_ added the code block

Reply:
Nice.